src/ConfigFiles/nativeLang.xml

```diff
@@ -22,7 +22,7 @@
 
 <GUP_NativeLangue name = "English">
 	<PopupMessages>
-		<MSGID_NOUPDATE>No update is available.</MSGID_NOUPDATE>
+		<MSGID_NOUPDATE>No update is available.\r\rThe auto-update may not have been activated yet.\rGo to the official website to check for a new version?</MSGID_NOUPDATE>

```

Comment:
As far as I understood, this is not used in Notepad++.
If I try to load this in debug, then the message is displayed incorrectly:
![image](https://user-images.githubusercontent.com/9664141/104957472-e5e81e00-59d6-11eb-9eda-5c567051238a.png)

It seems LF is not loaded properly from XML.